packages/paste-core/components/anchor/__tests__/anchor.test.tsx

```diff
@@ -7,7 +7,7 @@ describe('Anchor', () => {
   it('it should render an anchor', (): void => {
     const tree = renderer
       .create(
-        <Theme.Provider>
+        <Theme.Provider theme="console">
```

Comment:
Prevents all the snapshots needing to be updated, plus it's good practice to set the theme